simpledotorg/simple-server
# Add coverage percentage to Hypertension Indicators table on the dashboard

**Story card:** [sc-9941](https://app.shortcut.com/simpledotorg/story/9941/include-percentage-of-registration-beside-cumulative-registration-in-dashboard)

## Because

West Bengal has requested that on the dashboard we are able to show the percentage of patients registered against the total hypertensive population to enable CVHOs immediately identify how many more patients of the estimated hypertension population need to be reached. 

## This addresses

Adds a column in the Hypertension Indicators table on the Report page for states and organisations with % registration coverage across the region's estimated hypertensive population.

## Test instructions

- Go to a state/organization's hypertension report page
- Add estimated populations
- Check the Hypertension Indicators table for the coverage %s split across subregions
